neuro_data/pipeline_management/NeuroDataPipelineManagement.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+import datajoint as dj
+
+
+from neuro_data.static_images.data_schemas import StaticScanCandidate, StaticScan, ConditionTier, Frame, InputResponse, Eye, Treadmill, StaticMultiDataset, StaticMultiDatasetGroupAssignment, ExcludedTrial
+
+pipeline_anatomy = dj.create_virtual_module('pipeline_anatomy', 'pipeline_anatomy')
```

Comment:
first part of the create virtual module can only be 'anatomy' or 'fuse' or 'stimulus'

Reply:
I tend to prefer the full name of the schema, we have IDEs so they mostly auto correct for us.

Reply:
sure